TombEngine/Objects/TR3/Entity/tr3_claw_mutant.cpp

```diff
@@ -148,7 +148,7 @@ namespace TEN::Entities::Creatures::TR3
 	{
 		const auto& creature = *GetCreatureInfo(&item);
 
-		int plasmaBall = CreateNewEffect(item.RoomNumber);
+		int plasmaBall = CreateNewEffect(item.RoomNumber, ID_PROJ_BOMB, item.Pose);
```

Comment:
Why object ID is reassigned immediately after `CreateNewEffect`? Here it is `ID_PROJ_BOMB`, later it becomes `ID_ENERGY_BUBBLES`.